### PR TITLE
added php72 travis

### PR DESCRIPTION
**Pull request by kokspflanze**
added php72 travis in build matrix

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +38,15 @@ matrix:
     - php: 7.1
       env:
         - DEPS=latest
+    - php: 7.2
```
Comment (webimpress) [inline .travis.yml]: IMHO, PHP 7.2 should be added instead of "nightly"

Comment (xtreamwayz) [inline .travis.yml]: It has 7.2 AND nightly. Nightly refers to 7.3 at the moment. Not sure if 7.3 is already a travis php tag.

Comment (webimpress) [inline .travis.yml]: Oh, if it refers to PHP 7.3 it's good to have then !

Comment (xtreamwayz) [inline .travis.yml]: https://travis-ci.org/zendframework/zend-servicemanager/jobs/266571573#L267

**Pull request closed without merging**